core/common/src/test/kotlin/com/google/samples/apps/nowinandroid/core/result/ResultKtTest.kt

```diff
@@ -25,7 +25,7 @@ import kotlin.test.assertEquals
 class ResultKtTest {
 
     @Test
-    fun Result_catches_errors() = runTest {
+    fun result_catches_errors() = runTest {
```

Comment:
nit: ktlint does not understand that we are referencing actual class names here. I think we should keep them. (same for the next files)

Reply:
Shall we follow the `given_when_then` or `when_then` format?

Reply:
Or use `ktlint_function_naming_ignore_when_annotated_with` config for now and rename later if needed.

Reply:
There are other instances of this in other test files, that's why I recommended to use the .editorconfig parameter :)

Reply:
Please review: 56fbe13b5abe497df8b32a262001a922f3f9921c